auth-api/src/auth_api/models/affidavit.py

```diff
@@ -60,3 +61,12 @@ def find_pending_by_user_id(cls, user_id: int):
     def find_approved_by_user_id(cls, user_id: int):
         """Find pending affidavit by user id."""
         return cls.query.filter_by(user_id=user_id, status_code=AffidavitStatus.APPROVED.value).one_or_none()
+
+    @classmethod
+    def find_effective_by_user_guid(cls, user_guid: str):
+        """Find pending affidavit by user id."""
+        return db.session.query(Affidavit)\
+            .join(User, User.id == Affidavit.user_id)\
+            .filter(Affidavit.status_code.in_([AffidavitStatus.PENDING.value, AffidavitStatus.APPROVED.value]))\
```

Comment:
Do we need to get Affidavit by REJECTED as well since we need it while opening a rejected task in staff dashboard ?